web/src/pages/index.js

```diff
@@ -125,7 +125,11 @@ function HeroCodeExample() {
 
   auth: { /* full-stack auth out-of-the-box */
     userEntity: User,
-    methods: [ UsernameAndPassword ],
+    externalAuthAssociationEntity: SocialLogin,
```

Comment:
You are right, the existing name is a bit long. That was my biggest concern in this update PR, to be honest. :) I do like `externalAuthEntity`. Does that work for you @Martinsos? Mainly want everyone to agree since updating it is a lot of changes, so want to do it just once haha Good suggestion though

Reply:
Works for me!

Reply:
Ok, great! I will rename it to `externalAuthEntity`. 🥳

Reply:
I changed the name. I did leave the required opposite-end relationship column name as is, though (`externalAuthAssociations`):

```sql
entity User {=psl
    id                        Int           @id @default(autoincrement())
    username                  String        @unique
    password                  String
    externalAuthAssociations  SocialLogin[] 👈
psl=}
```